demo/admin/src/products/future/generated/ProductsGrid.tsx

```diff
@@ -97,9 +97,10 @@ type Props = {
     toolbarAction?: React.ReactNode;
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     rowAction?: (params: GridRenderCellParams<any, GQLProductsGridFutureFragment, any>) => React.ReactNode;
+    actionsColumnWidth?: number;
 };
 
-export function ProductsGrid({ filter, toolbarAction, rowAction }: Props): React.ReactElement {
+export function ProductsGrid({ filter, toolbarAction, rowAction, actionsColumnWidth = 52 }: Props): React.ReactElement {
```

Comment:
Why is this 52, and not 116? Because of the custom action?

Reply:
In this case only one of the comet-actions is shown (either the edit-button or the more-menu), so `52` is the default value, which is correct for when only one item is shown. 

When custom actions are passed in manually (using the `rowAction` prop), the width also needs to be set manually (using the `actionsColumnWidth` prop) as comet cannot know how many actions or what kind of elements are passed in.